theme/index.hbs

```diff
@@ -6,10 +6,15 @@
         <title>{{ title }}</title>
         {{#if is_print }}
         <meta name="robots" content="noindex">
+        <meta name="googlebot" content="noindex">
         {{else}}
         <meta name="robots" content="index, follow">
+        <meta name="googlebot" content="index, follow">
         {{/if}}
 
+        <meta name="keywords" content="SlimeVR, Full, Body, Tracking, Tracker, Trackers, FBT, Virtual, Reality, VR, open, source, DIY, Documentation, Guide">
+        <link rel="canonical" href="https://docs.slimevr.dev/">
```

Comment:
Cannonical should be defferent for each pages

Reply:
just remove it for now. We will see what the google search console says

Reply:
Canonical removed. Google does not use keyword meta like other search engines.

Reply:
But canonical should be fine since we are telling the search engine to index the main (home) page anyways.